phraseapp/config_test.go

```diff
@@ -202,7 +202,7 @@ func TestConfigPath_ConfigFromEnvButNotExisting(t *testing.T) {
 		t.Fatalf("expect an error, got none")
 	}
 
-	expErr := `file "phraseapp_does_not_exist.yml" (given in PHRASEAPP_CONFIG) doesn't exist`
+	expErr := `file "phraseapp_does_not_exist.yml" (from PHRASEAPP_CONFIG environment variable) doesn't exist`
```

Comment:
this is certainly defined somewhere else (not sure where) but I guess it would sound better if it says `from the PHRASEAPP_CONFIG environment variable`.